packages/pluggableWidgets/datagrid-date-filter-web/src/ui/DatagridDateFilter.scss

```diff
@@ -0,0 +1,145 @@
+@import "~react-datepicker/src/stylesheets/datepicker";
+@import "DatePicker";
+@import "variables";
+
+@font-face {
+    font-family: "datagrid-filters";
+    src: url("src/ui/fonts/datagrid-filters.eot");
+    src: url("src/ui/fonts/datagrid-filters.eot") format("embedded-opentype"),
+        url("src/ui/fonts/datagrid-filters.woff2") format("woff2"),
+        url("src/ui/fonts/datagrid-filters.woff") format("woff"),
+        url("src/ui/fonts/datagrid-filters.ttf") format("truetype"),
+        url("src/ui/fonts/datagrid-filters.svg") format("svg");
+    font-weight: normal;
+    font-style: normal;
+}
+
+.filter-container {
+    display: flex;
+    flex-direction: row;
+    flex-grow: 1;
+
+    .filter-input {
+        border-top-left-radius: 0;
+        border-bottom-left-radius: 0;
+    }
+
+    .btn-calendar {
+      margin-left: 5px; //Review in atlas, the current date picker is also 5px
+    }
+}
+
+.filter-selector {
+    display: flex;
+    align-items: stretch;
+    flex-direction: column;
+    padding-left: 0;
+    padding-right: 0;
+
+    .filter-selector-content {
+        display: flex;
+        flex: 1;
+        align-self: flex-end;
+
+        .filter-selector-button {
+            padding: 8px;
+            border-top-right-radius: 0;
+            border-bottom-right-radius: 0;
+            border-right: none;
+
+            &:before {
+                display: flex;
+                justify-content: center;
+                width: 20px;
+                height: 20px;
+                padding-left: 4px; /* The font has spaces in the right side, so to align in the middle we need this */
+            }
+        }
+
+        .filter-selectors {
+            position: absolute;
+            width: max-content;
+            left: 0;
+            margin: 0 8px;
+            padding: 0;
+            background: $background-color;
+            z-index: 99;
+            border-radius: 8px;
+            border: 1px solid $border-color;
+            list-style-type: none;
+            -webkit-box-shadow: 0 3px 14px 2px rgba(0, 0, 0, 0.02), 0 8px 10px 1px rgba(0, 0, 0, 0.04),
+                0 5px 5px -3px rgba(0, 0, 0, 0.08);
+            -moz-box-shadow: 0 3px 14px 2px rgba(0, 0, 0, 0.02), 0 8px 10px 1px rgba(0, 0, 0, 0.04),
+                0 5px 5px -3px rgba(0, 0, 0, 0.08);
+            box-shadow: 0 3px 14px 2px rgba(0, 0, 0, 0.02), 0 8px 10px 1px rgba(0, 0, 0, 0.04),
```

Comment:
This box shadow doesnt look aligned with the  popup menu's
`box-shadow: 0 2px 20px 1px rgba(5, 15, 129, 0.05), 0 2px 16px 0 rgba(33, 43, 54, 0.08);`

Delete all the rest

Reply:
Done